lib/helper/Puppeteer.js

```diff
@@ -45,6 +45,7 @@ const consoleLogStore = new Console();
  *
  * This helper should be configured in codecept.json or codecept.conf.js
  *
+ * * `browser`: (optional, default: chrome) - select browser
```

Comment:
- select browser: chrome or firefox (experiemental, requires `puppeteer-firefox` package)

Reply:
Btw, I don't see where default value for `browser` is acutally set